### PR TITLE
[rescheduler] Fixed few issues in rescheduler

### PR DESCRIPTION
**Pull request by piosz**
1. Wait 2 minutes after rescheduler is created in order to
   make sure that all addons has a chance to start.
2. Operate on copy of a node (without taint) while doing simulations.
3. Correctly handle iteration over PodList.

### CODE REVIEW

File: rescheduler/rescheduler.go [hunk 1]
```diff
@@ -58,6 +58,10 @@ var (
 func main() {
 	glog.Infof("Running Rescheduler")
 
+	// Wait for cluster stabilization.
+	// TODO(piosz): figure our a better way of verifying cluster stabilization here.
+	time.Sleep(2 * time.Minute)
```
Comment (mwielgus) [inline rescheduler/rescheduler.go]: can you make it a flag?

File: rescheduler/rescheduler.go [hunk 2]
```diff
@@ -58,6 +58,10 @@ var (
...
+
 	flags.Parse(os.Args)
```
Comment (mwielgus) [inline rescheduler/rescheduler.go]: Flag parsing should be the first thing. It may affect glog.

File: rescheduler/rescheduler.go (hunk 1 above)
Comment (piosz) [inline rescheduler/rescheduler.go]: Done

File: rescheduler/rescheduler.go (hunk 2 above)
Comment (piosz) [inline rescheduler/rescheduler.go]: Done

**Comment by mwielgus** [comment]
lgtm

**Pull request merged**